script/bash/deploy.sh

```diff
@@ -245,8 +245,8 @@ for L2_CHAIN_ID in $L2_CHAIN_IDS_ARRAY; do
         echo "No Etherscan API key found for $L2_CHAIN_ID"
         exit 1
     fi
-    echo "Deploying L2 Adapter for $L2_RPC ($L2_CHAIN_ID)"
```

Comment:
Why does this deploy only the adapter now?

Reply:
It deploys both erc20 and adapter for l2s and only the adapter for l1

The `DeployOFT` script handles the chainId and deploys accordingly